docs/website/docs/getting-started/cluster-setup/kubernetes.md

```diff
@@ -4,61 +4,64 @@ sidebar_position: 1
 ---
 
 # Setting up a Kubernetes cluster
-*Note that this guide is only helpful in setting up a development environment; this setup is not recommended for a production environment.*
+
+## Introduction
+This guide is helpful in setting up a development environment intended to be used with `odo`; this setup is not recommended for a production environment.
+
+`odo` can be used with ANY Kubernetes cluster. However, this development environment will ensure complete coverage of all features (including advanced) of `odo`.
 
 ## Prerequisites
-* You have a Kubernetes cluster setup, this could for example be a [minikube](https://minikube.sigs.k8s.io/docs/start/) cluster.
-* You have admin privileges to the cluster, since the Operator installation is only possible with an admin user.
+* You have a Kubernetes cluster set up (such as [minikube](https://minikube.sigs.k8s.io/docs/start/))
+* You have admin privileges to the cluster
```

Comment:
Why do you think that the part about Operator installation should be removed? It was added to convey why the admin access is needed.